java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -1604,43 +1604,42 @@ public void fcall_binary_with_keys(String prefix) {
         assumeTrue(SERVER_VERSION.isGreaterThanOrEqualTo("7.0.0"), "This feature added in version 7");
 
         String key = "{" + prefix + "}-fcall_with_keys-";
+        GlideString binaryString =
+                gs(new byte[] {(byte) 0xFE, (byte) 0xEE, (byte) 0xEF, (byte) 252, (byte) 0});
         SingleNodeRoute route = new SlotKeyRoute(key, PRIMARY);
-        String libName = "mylib_with_keys";
-        GlideString funcName = gs("myfunc_with_keys");
+        String libName = "mylib_with_keys_" + prefix;
+        GlideString funcName = gs("myfunc_with_keys_" + prefix);
         // function $funcName returns array with first two arguments
         String code =
-                generateLuaLibCode(libName, Map.of(funcName.toString(), "return {keys[1], keys[2]}"), true);
+                generateLuaLibCode(libName, Map.of(funcName.toString(), "return {args[1]}"), true);
```

Comment:
please rename the test if we are no longer passing in keys

Reply:
We pass it, but don't use it